reference/src/client/include/sa_types.h

```diff
@@ -515,6 +515,29 @@ typedef struct {
     sa_key khmac;
 } sa_import_parameters_typej;
 
+/**
+ * Import parameters for a SOC key container. This structure is used to signal the SecApi compatability version of the
+ * key container. This structure is required for SecApi 2 key containers and is optional for SecApi 3 key containers.
+ * If the sa_key_import parameters field is NULL, a SecApi 3 key container is assumed. However, it is useful to always
```

Comment:
Recommend clarifying the usage of sa_key_import parameters field. Obviously not part of this structure, I think you are implying that a SecAPI2 Key Container must have a parameter with the sa_import_parameters_soc struct details in certain function calls. Should we add a unique header value to the struct to verify the correct data has been provided when the parameter is non-NULL?

Reply:
I for simplicity, I will say that use of the structure is required, but for backward compatibility it will not fail if omitted.